### PR TITLE
fix(titus): Fetch ipv4 and ipv6 from titus api

### PR DESCRIPTION
**Pull request by aravindmd**
Today we use `agentId` to infer the `instanceId` and make an additional call to `/instance` endpoint to get the `ipv6Address` from the `networkInterface`

This change is to get the `ipv4` and `ipv6` addresses from Titus API directly . `ipv4` address is transitioning from `titus.containerIp` to `task.containerIPv4`

Todo : once these changes are merged and things are tested , we can remove the agentId and containerIp fields

### CODE REVIEW

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy [hunk 1]
```diff
@@ -102,6 +107,14 @@ class TitusInstance implements Instance {
     agentId
   }
 
+  String getIpv4Address() {
+    ipv4Address
+  }
+
+  String getIpv6Address() {
+    ipv6Address
```
Comment (aravindmd) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy:115]: the earlier issue was due to this getter not being renamed with intellij refactor + groovy !

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy [hunk 2]
```diff
@@ -102,6 +107,14 @@ class TitusInstance implements Instance {
     agentId
   }
 
+  String getIpv4Address() {
+    ipv4Address
```
Comment (marchello2000) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy:111]: i personally hate the groovy return-less syntax, but for simple getters i am going to say it's just a nit, up to you

Comment (aravindmd) [reply clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/model/TitusInstance.groovy]: yep agree , added the return statement

**Comment by aravindmd** [comment]
not sure why i added the label and it ended up removing it , it was not showing the label to begin with

**Pull request merged**